mmv1/openapi_generate/parser.go

```diff
@@ -391,8 +391,13 @@ func writeObject(name string, obj *openapi3.SchemaRef, objType openapi3.Types, u
 		}
 
 		field.Type = "NestedObject"
-
-		field.Properties = buildProperties(obj.Value.Properties, obj.Value.Required)
+		if obj.Value.AdditionalProperties.Schema != nil {
+			field.Properties = buildProperties(obj.Value.AdditionalProperties.Schema.Value.Properties, obj.Value.AdditionalProperties.Schema.Value.Required)
+			field.KeyName = "TODO: CHANGEME"
+			field.KeyDescription = "TODO: CHANGEME"
```

Comment:
It looks like `KeyDescription` isn't used at all, so we probably don't need to populate it. Unless we want to change that?

https://github.com/GoogleCloudPlatform/magic-modules/blob/724a0dc6a2262433a64ad61b17ee01307085c1aa/mmv1/templates/terraform/nested_property_documentation.html.markdown.tmpl#L10